### ISSUE TITLE
Indicator menue -> System wrong entry

### ISSUE THREAD

**Issue by lutin11** [ubports/ubuntu-touch#1513]
Device: Nexus 5
Chanel: stable and devel

I'm not sure this is wondered or not but under the indicator menu -> Syestem I've got 5 list items
- About this device...
- Ubuntu Help....
- System Settings...
- Lock
- Shutdown...

Instead of 'Ubuntu Help...', that redirect to the Ubuntu Forum, I would expect to have an 'Ubport Help' (or Lorimi Help)

**Comment by gbdomubpkm** [comment]
I tried on OPO and BQ E5 HD  today last dev update : seems to be solved.
See :
![2020-07-29T10:07:21 273Z](https://user-images.githubusercontent.com/50362604/88787780-b62d2b80-d194-11ea-9dbf-bc57f576fbfe.png)

**Comment by Flohack74** [comment]
But "report a bug" is not translated... We are working on this now ^^

**Comment by gbdomubpkm** [comment]
Yes. For french, it can be for example : 'Signaler un problème', 'Signaler un bug'...
